TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/support/SocketConnectionObservable.java

```diff
@@ -4,60 +4,65 @@
 
 import com.novoda.tpbot.Result;
 
-import io.socket.client.Ack;
 import io.socket.client.Socket;
 import io.socket.client.SocketIOException;
 import io.socket.emitter.Emitter;
 
-import static io.socket.client.Socket.EVENT_CONNECT_ERROR;
+import static io.socket.client.Socket.*;
```

Comment:
Ah, this happens when you have more than three imports from the same package, it collapses when auto-formatting. I'll try and put them back in.